src/components/FlowStep/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from "react"
+import { Box, Icon, useColorModeValue } from "@chakra-ui/react"
+import { BodyMd, BodySm, LabelLg, LabelSm, LabelXs } from "../Typography"
+import { IoArrowForwardCircleSharp } from "react-icons/all"
+
+export type FlowStepStatus = "active" | "inactive" | "complete"
+export type FlowStepPosition = "vertical" | "horizontal"
+export type FlowStepSize = "sm" | "lg"
+
+export interface FlowStepProps {
+  status?: FlowStepStatus
+  position?: FlowStepPosition
+  size?: FlowStepSize
+  preTitle: string
+  title: string
+  description?: string | JSX.Element
+  hideDescription?: boolean
+  hideDescriptionArrow?: boolean
+}
+
+export const FlowStep: FC<FlowStepProps> = ({
+  status = "inactive",
+  position = "vertical",
+  size = "lg",
+  preTitle,
+  title,
+  description,
+  hideDescription = false,
+  hideDescriptionArrow = false,
```

Comment:
I think we usually keep specific naming convention for boolean values by adding `is` or `has` etc. at the beginning (might be wrong tho, but I'm pretty sure we have that naming convention in token-dashbaord). So i would propose to change it to:

```suggestion
  isDescriptionHidden = false,
  isDescriptionArrowHidden = false,
```